### PR TITLE
Enable profile-based authentication for EFS

### PR DESCRIPTION
**Pull request by wavewave**
As we continuously update this branch, EFS is also updated with profile parameter.

### CODE REVIEW

File: nixopsaws/resources/efs_common.py [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+            self._session = nixopsaws.ec2_utils.session(**{
+                "region_name": self.region,
+                "profile_name": profile,
+                "aws_access_key_id": access_key_id,
+                "aws_secret_access_key": secret_access_key,
+                "aws_session_token": os.environ.get('AWS_SESSION_TOKEN')
+            })
+
+        self._client = self._session.client('efs')
+                  #         region_name=region or self.region,
```
Comment (drostron) [inline nixopsaws/resources/efs_common.py]: stray comment

File: nixopsaws/resources/elastic_file_system.py [hunk 2]
```diff
@@ -107,6 +108,7 @@ def create(self, defn, check, allow_reboot, allow_recreate):
 
         def tag_updater(tags):
             # FIXME: handle removing tags.
+            print ("self.fs_id: {0}".format(self.fs_id))
```
Comment (drostron) [inline nixopsaws/resources/elastic_file_system.py]: should this stick around?

**Review by drostron — COMMENTED** [review]
Thanks! Good to see some direction in options handling.

File: nixopsaws/resources/efs_common.py (hunk 1 above)
Comment (wavewave) [reply nixopsaws/resources/efs_common.py]: removed. thanks!

File: nixopsaws/resources/elastic_file_system.py (hunk 2 above)
Comment (wavewave) [reply nixopsaws/resources/elastic_file_system.py]: nope. removed.

**Review by drostron — APPROVED** [review]

**Pull request merged**